Algorithmia/client.py

```diff
@@ -40,6 +41,11 @@ def dir(self, dataUrl):
         if dataUrl.startswith('file://'): return LocalDataDirectory(self, dataUrl)
         else: return DataDirectory(self, dataUrl)
 
+    # Used to send insight data to Algorithm Queue Reader in cluster
+    def report_insights(self, insights):
+        return Insights(insights)
```

Comment:
Two comments:
1) why are we returning theInsights object? Seems like void return type makes more sense here.
2) I think this should be wrapped in a try/catch so make local running easier. Right now I think it will throw exception if running locally with insights.